tests/transformers/test_transformers_model_export.py

```diff
@@ -3653,3 +3653,81 @@ def test_whisper_model_supports_timestamps(raw_audio_file, whisper_pipeline):
     first_timestamp = prediction["chunks"][0]["timestamp"]
     assert isinstance(first_timestamp, tuple)
     assert prediction_inference["chunks"][0]["timestamp"][1] == first_timestamp[1]
+
+
+def test_pyfunc_model_log_load_with_artifacts_snapshot(tmp_path):
+    snapshot_location = snapshot_download(
+        repo_id="prajjwal1/bert-tiny",
+        local_dir=tmp_path.joinpath("bert-tiny"),
+        # to avoid tmpdir OSError: [Errno 30] Read-only file system
+        local_dir_use_symlinks=False,
+    )
+
+    tokenizer = transformers.AutoTokenizer.from_pretrained(snapshot_location)
+    model = transformers.MobileBertForQuestionAnswering.from_pretrained(snapshot_location)
```

Comment:
I am surprised we can load a BERT checkpoint into `MobileBertForQuestionAnswering`... let's change it to `BertForQuestionAnswering ` for clarity.